falcon/testing/base.py

```diff
@@ -26,14 +26,16 @@
 
 
 class TestBase(unittest.TestCase):
-    """Extends ``testtools.TestCase`` to support WSGI integration testing.
+    """Extends ``testtools.TestCase`` to support WSGI functional testing.
```

Comment:
I know this is a little out of scope of discussion, but isn't this extends testtools.TestCase or unittest.TestCase? As a side note, I do appreciate us not forcing people to use testtools. I believe if we're staying with a unittest compatible approach, being neutral and allowing the implementer to choose is important.